### ISSUE TITLE
Falling Through World

### ISSUE THREAD

**Issue by Crylawns** [ccshiro/cc-buglist#1090]
I logged out in the middle of a battleground for about 5 minutes with my character alive. Upon logging back in I was returned to the battleground (WSG), and I proceeded to fall through the world where I died in the Ruins of Alterac. Try standing at the Horde side spawn in WSG while logging back in to reproduce.

**Comment by hddist** [comment]
Confirmed. 
Had the same issue with a few other folks, straight after WSG had finished you get ported to Alterac Mountains for no reason, fall through World for a few seconds, then appear dead in the Ruins fo Alterac.

(My Assumption is: maybe if you were in combat, casting something etc right in the moment when WSG finishing you get this bugged).

Instead of releasing the spirit at the Ruins of Alterac I have decided to go to Tarren Mill (Hillsbrad Foothills), found a spirit healer there, accepted the ressurection, but got ported back (alive) to Alterac Mountains graveyard. 

![wowscrnshot_080815_023949](https://cloud.githubusercontent.com/assets/13605723/9148651/e578521e-3d7b-11e5-8356-a0f8416f4b74.jpg)
